applications/minotari_console_wallet/src/cli.rs

```diff
@@ -166,123 +163,49 @@ pub struct BurnMinotariArgs {
 }
 
 #[derive(Debug, Args, Clone)]
-pub struct FaucetCreateKeyPairArgs {
-    #[clap(long)]
-    pub key_branch: String,
-}
-
-#[derive(Debug, Args, Clone)]
-pub struct FaucetCreateAggregateSignatureUtxoArgs {
-    #[clap(long)]
-    pub amount: MicroMinotari,
+pub struct FaucetGenerateSessionInfoArgs {
     #[clap(long)]
     pub fee_per_gram: MicroMinotari,
     #[clap(long)]
-    pub n: u8,
-    #[clap(long)]
-    pub m: u8,
+    pub commitment: String,
     #[clap(long)]
-    pub message: String,
+    pub output_hash: String,
     #[clap(long)]
-    pub maturity: u64,
+    pub recipient_address: TariAddress,
     #[clap(long)]
```

Comment:
```suggestion
    #[clap(long, default_value=false)]
```
So we dont have to add it every time

Reply:
It is switch, therefor already default false